src/js/components/common/dialogs/physicaldeviceonboarding.js

```diff
@@ -133,7 +145,9 @@ export class PhysicalDeviceOnboarding extends React.Component {
 const actionCreators = { findLocalIpAddress, setOnboardingApproach, setOnboardingDeviceType };
 
 const mapStateToProps = state => {
+  const docsVersion = state.app.docsVersion ? `${state.app.docsVersion}/` : 'development/';
```

Comment:
Why do you want to version this page? Wouldn't be ok to just point to latest stable release? This way if we add images in the next release, we still show them for an old running server.

Reply:
Mostly because we can't be certain that the latest image versions will always be compatible with the version the user is currently onboarding in (or if they will exist there at all in later versions)...
But I see the benefit of having it just point to the current version... any preference?

Reply:
Ok. Better safe than sorry (i.e. leave it as is)